src/compiler/c.rs

```diff
@@ -77,7 +79,7 @@ struct CCompilation<I: CCompilerImpl> {
     parsed_args: ParsedArguments,
     executable: String,
     /// The output from running the preprocessor.
-    preprocessor_output: Vec<u8>,
+    preprocessor_result: process::Output,
```

Comment:
This change ought to make it easier to fix https://github.com/mozilla/sccache/issues/72 as well!